assets/src/js/frontend/give-donations.js

```diff
@@ -510,35 +510,29 @@ Give.form = {
 			return price_id;
 		},
 
-		/**
-		 * Get form minimum amount
-		 *
-		 * @since 1.8.17
-		 * @param {object} $form
-		 *
-		 * @return {string}
-		 */
-		getMinimumAmount: function( $form ) {
-			return Give.fn.unFormatCurrency(
-				$form.find( 'input[name="give-form-minimum"]' ).val(),
-				Give.form.fn.getInfo( 'decimal_separator', $form )
-			);
-		},
-
-		/**
-		 * Get form maximum amount
-		 *
-		 * @since 2.1
-		 * @param {object} $form
-		 *
-		 * @return {string}
-		 */
-		getMaximumAmount: function( $form ) {
-			return Give.fn.unFormatCurrency(
-				$form.find( 'input[name="give-form-maximum"]' ).val(),
-				Give.form.fn.getInfo( 'decimal_separator', $form )
-			);
-		},
+        /**
+         * Get form minimum amount
+         *
+         * @since 1.8.17
+         * @param {object} $form
+         *
+         * @return {string}
+         */
+        getMinimumAmount: function ($form) {
+            return $form.find('input[name="give-form-minimum"]').val();
```

Comment:
@raftaar1191 what was the issue? The min/max field is already unformatted?

Reply:
@DevinWalker before we used to pass the min and max value in a formatted way but from 2.1 we are not doing that 

If you want I can add the PR link too where we remove the PHP formatting for min and max value

Reply:
Issues link: https://github.com/WordImpress/Give/issues/3029 where we remove the formatting from PHP

Reply:
Thanks - PR for ref: https://github.com/WordImpress/Give/pull/3148/files

Reply:
@DevinWalker @raftaar1191 it had been already fixed but merge into the wrong pr. This happened because for some time we set `2.2` as default branch (I already fixed it ).

for ref: https://github.com/WordImpress/Give/pull/3157/files